packages/core/src/DdSdkReactNativeConfiguration.tsx

```diff
@@ -8,30 +8,43 @@ import type { ProxyConfiguration } from './ProxyConfiguration';
 import type { SdkVerbosity } from './SdkVerbosity';
 import { TrackingConsent } from './TrackingConsent';
 
+const DEFAULTS = {
+    nativeCrashReportEnabled: false,
+    sessionSamplingRate: 100.0,
+    resourceTracingSamplingRate: 20.0,
+    site: 'US',
```

Comment:
I think we can use `US1`, since DCs without digits were deprecated long time ago in native SDKs